lib/core.mli

```diff
@@ -667,31 +617,30 @@ module Children : sig
 
       let map element mapper = map_internal children_internal element mapper]
 
-  val map_with_index : element list -> (element -> int -> element) -> element
+  val mapi : element list -> (element -> int -> element) -> element
     [@@js.custom
-      val map_with_index_internal :
+      val mapi_internal :
```

Comment:
I got some name conflicts when I used the same name for some of these internal functions, and so I did some experiments to try to figure out what gen_js_api would actually generate, and in what way they "leak". I can't remember exactly what I concluded, but I think these function names leak out into the file namespace. So I started to think about how to minimize the noise this creates, that prefixing them with `_` prevent them from showing up in completion menus, as they would when suffixed with `_internal`, for example. It's also easier to rename prefixed names (in vim). It's a bit of a half-baked experiment, but so far I can't see any downsides to prefixing with `_` rather than suffixing with `_internal` at least.